app/models/tank.rb

```diff
@@ -5,4 +5,8 @@ class Tank < ApplicationRecord
   has_many :measurements, through: :measurement_events
 
   has_one :family, required: false
+
+  def empty?
```

Comment:
I'm worried about this because `empty?` is a ruby/rails word in of itself. Maybe a synonym would be better?

Reply:
This is an excellent point, Jennifer and I's expectation is that "empty?" is a word we expect Ruby/Rails folks to reach for when attempting to decide if the Tank is empty. _However_ I could see that being an issue if people are using `empty?` as an alternative for `blank?` or `!present?` 

I see `empty?` as useful for collections or things that are not nil, but may not have any contents; while `blank?` is useful for 'this is nil _or_ empty'

```
[].nil? # => false
[].blank? # => true
[].empty? # => true
"".blank?  # => true
"".empty? #=>true
```

LMK what you want us to do, we can take care of it next week! :)